src/ProfilesPage/ProfilePage.js

```diff
@@ -86,6 +92,30 @@ class ProfilePage extends Component {
     this.setState({editUserEnable: editable});
   }
 
+  onSuspend() {
+    const id = this.props.userInfo.id
```

Comment:
destructure props used within method i.e. `const { userInfo, suspendUser } = this.props;`

Reply:
done